docarray/index/backends/hnswlib.py

```diff
@@ -188,6 +183,12 @@ class RuntimeConfig(BaseDocIndex.RuntimeConfig):
             }
         )
 
+    @dataclass
+    class RuntimeConfig(BaseDocIndex.RuntimeConfig):
+        """Dataclass that contains all "dynamic" configurations of HnswDocumentIndex."""
+
+        pass
```

Comment:
I believe there are things that we could put here, but that is for a different PR I suppose